Add usage sentence for both building and endpoint file. (#2102)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/turning-on-and-using-call-quality-dashboard.md

<a name="BKMKTenantDataFile"> </a>

### Building data file

> CQD use Building data file by first derive Subnet column from expanding Network+NetworkRange column, then joining Subnet column to call record’s First Subnet/Second Subnet column to show Building/City/Country/Region… information. The format of the data file you upload must meet the following to pass the validation check before uploading.

Reviewer: ```suggestion
CQD uses a Building data file derived from a Subnet column (expanding Network+NetworkRange) and the call record’s First Subnet/Second Subnet column to show Building/City/Country/Regional information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```

Author: changed use to uses, prefer do not change other stuff due to:
1. There are multiple building data files
2. Subnet column is derived from expanding Network+NetworkRange, 2 columns in building data file(s).
3. The column is called region, not regional.

> The network range can be used to represent a supernet (combination of several subnets with a single routing prefix). All new building uploads will be checked for any overlapping ranges. If you have previously uploaded a building file, you should download the current file and re-upload it to identify any overlaps and fix the issue before uploading again. Any overlap in previously uploaded files may result in the wrong mappings of subnets to buildings in the reports. Certain VPN implementations do not accurately report the subnet information. It is recommended that when adding a VPN subnet to the building file, instead of one entry for the subnet, separate entries are added for each address in the VPN subnet as a separate 32-bit network. Each row can have the same building metadata. For example, instead of one row for 172.16.18.0/24, you should have 256 rows, with one row for each address between 172.16.18.0/32 and 172.16.18.255/32, inclusive. 

### Endpoint data file

> CQD use Endpoint data file by joining its EndpointName column to call record’s First Client Endpoint Name/Second Client Endpoint Name column to show Endpoint Make/Model/Type information. The format of the data file you upload must meet the following to pass the validation check before uploading.

Reviewer: ```suggestion
CQD uses an Endpoint data file derived from joining the EndpointName column to the call record’s First Client Endpoint Name/Second Client Endpoint Name column to show Endpoint Make/Model/Type information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```

Author: Change use to uses and the in front of call records, prefer don't change other stuff due to:
EndpointName is part of the Endpoint data file, no derive.

Reviewer: Thanks, we edit for language and grammar and  it's completely your call to preserve the technical meaning.
